WooCommerce/Classes/ViewRelated/Dashboard/Settings/Settings/SettingsViewModel.swift

```diff
@@ -226,14 +226,6 @@ private extension SettingsViewModel {
         let storeSettingsSection: Section? = {
             var rows: [Row] = []
 
-            if !featureFlagService.isFeatureFlagEnabled(.paymentsHubMenuSection) {
-                if upsellCardReadersAnnouncementViewModel.shouldBeVisible {
-                    rows.append(.upsellCardReadersFeatureAnnouncement)
```

Comment:
I believe we still want to keep the upsell announcement logic?

Reply:
Not really, since there is no IPP Settings entry anymore, we decided that the banner doesn't make too much sense in settings https://github.com/woocommerce/woocommerce-ios/issues/7364

Reply:
That makes sense, thanks for the context